apps/mobile/src/app/wallet/(tabs)/collectibles.tsx

```diff
@@ -1,9 +1,23 @@
 import { StyleSheet, Text, View } from 'react-native';
 
+import { APP_ROUTES } from '@/constants';
```

Comment:
Unrelated to PR but, to align with extension, can the alias be sans `/`?  So `@constants`?